workers/c_client_direct/client.c

```diff
@@ -129,6 +137,16 @@ int main(int argc, char** argv) {
 
   /* Connect to SpatialOS. */
   Worker_ConnectionParameters params = Worker_DefaultConnectionParameters();
+  params.network.connection_type = WORKER_NETWORK_CONNECTION_TYPE_MODULAR_KCP;
+  params.network.modular_kcp.security_type = 0;
+  Worker_KcpTransportParameters kcp_parameters;
```

Comment:
These don't need to be explicitly set to defaults - this is already done by `Worker_DefaultConnectionParameters()` - so you can omit lines 142 to 149 completely.

If you did want to customise a value for both upstream and downstream, it could still be done with:
```
params.network.modular_kcp.upstream_kcp.foo = ...;
params.network.modular_kcp.upstream_kcp.bar = ...;
params.network.modular_kcp.downstream_kcp = params.network.modular_kcp.upstream_kcp;
```
Where you don't need to mention the defaulted values (although you still could if you wanted to be explicit about what is customisable), and where the last line copies the upstream to downstream to avoid having to set everything twice.

One problem with the existing approach in this PR is that if a new field is added to `Worker_KcpTransportParameters`, this code will leave it with an indeterminate value, and possibly lead to random behaviour, crash, etc. Perhaps the Worker SDK should expose a `Worker_DefaultKcpTransportParameters()` to help avoid that.

Reply:
Removed the mentioned lines. I guess I don't need to add anything in place? PTAL.